src/translations/nb/referenceTypes.json

```diff
@@ -0,0 +1,29 @@
+{
+  "book": "Bok",
+  "chapter": "Kapittel",
+  "degree": "Grad",
+  "journalArticle": "Publikasjon i tidsskrift",
+  "report": "Rapport",
+  "subtypes_report": {
+    "report": "Rapport",
+    "researchReport": "Forskningsrapport",
+    "policyReport": "Policyrapport",
+    "workingPaper": "Arbeidsnotat"
+  },
+  "subtypes_journal_article": {
+    "article": "Artikkel",
+    "editorial": "Leder",
+    "letter": "Brev til redaktør",
+    "review": "Anmeldelse",
+    "shortCommunication": "Short communication"
```

Comment:
Very Norwegian, yes

Reply:
sånn er det bare :-)

Reply:
så vi har faktisk ikke noe norsk for dette?

Reply:
stemmer